sentry-spring-boot-starter/src/test/kotlin/io/sentry/spring/boot/it/SentrySpringIntegrationTest.kt

```diff
@@ -137,6 +142,17 @@ class SentrySpringIntegrationTest {
             }, anyOrNull())
         }
     }
+
+    @Test
+    fun `does not send events for handled exceptions`() {
+        val restTemplate = TestRestTemplate().withBasicAuth("user", "password")
+
+        restTemplate.getForEntity("http://localhost:$port/throws-handled", String::class.java)
+
+        await.during(Duration.ofSeconds(2)).untilAsserted {
+            verifyZeroInteractions(transport)
+        }
```

Comment:
wouldn't be better to `verify` the `Hub` instead which is a sync call and you wouldn't need to wait up to 2s?

Reply:
Good idea!